modules/mod_acl_user_groups/mod_acl_user_groups.erl

```diff
@@ -115,6 +115,7 @@ event(#postback{message={delete_all, Args}}, Context) ->
             z_render:wire({alert, [{message, ?__("Delete is canceled, there are users in the user groups.", Context)}]}, Context)
```

Comment:
Line 115 is too long:             z_render:wire({alert, [{message, ?__("Delete is canceled, there are users in the user groups.", Context)}]}, Context).